enterprise/BUILD

```diff
@@ -11,7 +11,6 @@ filegroup(
     visibility = ["//visibility:public"],
 )
 
-# gazelle:ignore
```

Comment:
This prevents Gazelle from modifying or formatting the entire build file. It seems this is not necessary here.